src/test/resources/scripts/schema.sql

```diff
@@ -0,0 +1,16 @@
+CREATE SCHEMA IF NOT EXISTS MA_STUDENT;
+
+CREATE SEQUENCE IF NOT EXISTS CUSTOMER_SEQ
```

Comment:
Haven't sence. You didn't need that, becouse you havent Id generator in Customer entity